contribs/application/src/main/java/org/matsim/application/analysis/TravelTimeAnalysis.java

```diff
@@ -39,13 +42,19 @@
     @CommandLine.Parameters(arity = "1", paramLabel = "INPUT", description = "Input run directory")
     private Path runDirectory;
 
+    @CommandLine.Option(names = "--analysis-type", defaultValue = "post-analysis", description = "type of the analysis. Choose from [pre-analysis, post-analysis]", required = true)
+    private String type;
```

Comment:
You could add an enum for this option, the cli parser will handle it automatically.

Reply:
May I know how to make it handle the enum automatically?

Reply:
Just create one and use it here instead of String. That should be all that is needed to do.